deep_agent/aegra/lifecycle.py

```
@@ -0,0 +1,680 @@
+                    (POD_ID, run_id),
+                )
+                updated = bool(cur.rowcount > 0)
+            conn.commit()
+
+        if updated:
+            logger.info(
+                "Expired lease for run %s (started_by=%s) — reaper will recover",
+                run_id,
+                POD_ID,
+            )
+        return updated
```

Ruff (0.15.21)

[warning] Consider moving this statement to an `else` block

(TRY300)